### PR TITLE
Documents the raw flag

### PR DESCRIPTION
**Pull request by adriancole**
A raw flag was introduced recently, and exists for any recent zipkin server

### CODE REVIEW

**Comment by adriancole** [comment]
cc @abesto jic you're not watching this

File: zipkin-api.yaml [hunk 1]
```diff
@@ -139,12 +140,24 @@ paths:
         required: true
         description: the 64-bit hex-encoded id of the trace as a path parameter.
         type: string
+      - name: raw
```
Comment (mohsen1) [inline zipkin-api.yaml]: `required` is required. 

I guess it would be a good idea to print validation errors in `npm test`.

Comment (adriancole) [inline zipkin-api.yaml]: thanks! will add that. yeah good idea, esp if hooked up to travis

Comment (adriancole) [inline zipkin-api.yaml]: done!

Comment (mohsen1) [inline zipkin-api.yaml]: Made https://github.com/openzipkin/zipkin-api/pull/7 to have better validation.

Comment (adriancole) [inline zipkin-api.yaml]: thanks! I was mucking around with the old one.. seems result is no longer
defined when JSON is ok. Will rebase on yours.

Comment (adriancole) [inline zipkin-api.yaml]: rebased!

**Comment by adriancole** [comment]
the green color is thanks to @mohsen1

**Comment by mohsen1** [comment]
:+1:

**Comment by abesto** [comment]
@adriancole I wasn't but now I am (watching the repo, that is). Thanks for the mention!

**Pull request merged**